### ISSUE TITLE
Feature: Title needs to be limited to 49 characters.

### ISSUE THREAD

**Issue by bdougie** [open-sauced/insights#1192]
### Type of feature

🍕 Feature

### Current behavior

Today the title for a highlight is unlimited.

![ezgif com-video-to-gif (2)](https://github.com/open-sauced/insights/assets/5713670/0686390c-9d50-46c3-9f71-cc20023a2932)

### Suggested solution

Add a limit of 49 characters. If a highlight is submitted with a title longer than 49 characters, we should display a toast error message stating, "Title must be less than 49 characters."

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [ ] I agree to follow this project's Contribution Docs

**Comment by a0m0rajab** [comment]
.take

**Comment by takanome-dev** [comment]
> I am a bit confused here since the auto-take workflow worked and at the same time the bot said it's not working.

I think we re-enabled it and didn't update the message.

**Comment by a0m0rajab** [comment]
Thank you for the information!

**Comment by bdougie** [comment]
Yeah. We need to remove the messsage.

**Comment by takanome-dev** [comment]
> Yeah. We need to remove the messsage.

How about updating the action to look for some labels and show a message according to the labels? 

For instance, if an issue has a label `members-only` folks not in the team will get a similar message as above(and won't be assigned). Otherwise, they will get the default message ->  https://github.com/open-sauced/open-sauced/blob/main/.github/workflows/triage.yml

THIS IS PROBABLY NOT NEEDED IF ALL ISSUES ARE UP FOR GRABS (or for some other reasons:sweat_smile:)!

**Comment by a0m0rajab** [comment]
@takanome-dev your suggestion is possible to be done with this: 
```yml
if: contains(github.event.issue.labels.*.name, 'bug')
```
here is the docs: https://docs.github.com/en/actions/learn-github-actions/expressions#example-using-an-object-filter
The discussion: https://github.com/orgs/community/discussions/27722

**Comment by takanome-dev** [comment]
What's your take @bdougie?

**Comment by bdougie** [comment]
I think this conversation is outside the scope of this issue and should move to a new issue or the discord.

I also think we are looking for a solution to a different problem than what was presented. We can remove the message for now. It was added while we did not have a lot of structure in this project. Anything in the Papercuts milestone are ups for grabs. Not sure if adding a Papercuts label is the solution.
